.github/workflows/forge-build.yml

```diff
@@ -5,8 +5,8 @@ on:
     inputs:
       cache-path:
         default: |
+          ~/.bun/install/cache
           cache
-          node_modules
```

Comment:
We need to cache `node_modules` as well for prettier and lint caches.

Reply:
Welp, this is a problem.

caching both `~/.bun` and `node_modules` would result into double caching.

We need to set up dedicated folders for the linting cache, and add them to the git ignore files.